go.sum

```diff
@@ -27,6 +25,8 @@ github.com/stretchr/objx v0.1.0 h1:4G4v2dO3VZwixGIRoQ5Lfboy6nUhCyYzaqnIAPPhYs4=
 github.com/stretchr/objx v0.1.0/go.mod h1:HFkY916IF+rwdDfMAkV7OtwuqBVzrE8GR6GFx+wExME=
 github.com/stretchr/testify v1.5.1 h1:nOGnQDM7FYENwehXlg/kFVnos3rEvtKTjRvOWSzb6H4=
 github.com/stretchr/testify v1.5.1/go.mod h1:5W2xD1RspED5o8YsWQXVCued0rvSQ+mT+I5cxcmMvtA=
+github.com/urso/antlr4/runtime/Go/antlr v0.0.0-20220707133017-7c6fad5e483f h1:9FKNRsjztjxjsYJ4zCevMBZhdQRLVjIfJtMFnF8RGm0=
+github.com/urso/antlr4/runtime/Go/antlr v0.0.0-20220707133017-7c6fad5e483f/go.mod h1:daxHKs6tlEdQ4bDNRiP/WnDGeopYrRWO617cvizdDgY=
```

Comment:
Looking at this again, it seems we should have pinned this repo to https://github.com/antlr/antlr4/releases/tag/4.7.2, instead of 4.9.3, right?

Reply:
Why? AFAIK 4.9.3 is backwards compatible to 4.7.2. It is 4.10 that changes the internal 'format' and APIs. I can update the repo to pin it to 4.7.2 if you think that would help.

Reply:
I'm investigating a data race that traces down to antlr4, so sticking to the version we were using before would rule out this change

Reply:
Good point. Will update my repo to 4.7.2